pkg/util/util.go

```diff
@@ -0,0 +1,187 @@
+//
```

Comment:
A number of [changes](https://github.com/sigstore/rekor/pull/1071/files#diff-05d01822b637dcfaa33f0867a1a327fe69bd9e8038c54b8e0ced6711a121b595) were made to tests/util.go to support code coverage that you'll want to add here.

Mainly you need to update all of the `run` functions to prepend the coverage flag and then strip the coverage info off of the output before returning it. You'll also need to update the coverage flag to use /tmp/pkg-rekor-cli... so that the correct files are merged.

Reply:
I have updated it. Can you please check it and let me know? Thanks

Reply:
You copied over the coverageFlag func but aren't actually calling it anywhere. You just need to make sure it's added as the first arg before exec is called.

Reply:
Fixed it.